students/10th/byungjunkim/instagram/post/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from user.models import User
+
+
+class Post(models.Model):
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    title= models.CharField(max_length=128)
+    text = models.CharField(max_length=1024,blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class meta:
+        db_table = 'Post'
+
+class Comment(models.Model):
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    post = models.ForeignKey(Post,on_delete=models.CASCADE)
+    text = models.CharField(max_length=128)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class meta:
+        db_table='Comment'
+
+class Thumbs(models.Model):
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    post = models.ForeignKey(Post,on_delete=models.CASCADE)
+    class meta:
```

Comment:
class 선언하기 전에 한줄 추가해주세요.
```suggestion

    class Meta:
```